ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -29,10 +30,18 @@ const useCurrentConfirmation = () => {
   const latestPendingApproval = useSelector(latestPendingConfirmationSelector);
   const confirmationId = paramsConfirmationId ?? latestPendingApproval?.id;
 
-  const redesignedConfirmationsEnabled = useSelector(
+  const isRedesignedConfirmationsUserSettingEnabled = useSelector(
     getRedesignedConfirmationsEnabled,
   );
 
+  const isRedesignedConfirmationsFeatureEnabled = useSelector(
+    getIsRedesignedConfirmationsFeatureEnabled,
+  );
+
+  const isRedesignedConfirmationsDeveloperSettingEnabled =
+    process.env.ENABLE_CONFIRMATION_REDESIGN ||
```

Comment:
The toggle UI also relies on this selector which displays as selected even if the preferences is false and the env var is true. Hence this was placed here instead of the selector.